build.gradle

```diff
@@ -5,7 +5,7 @@ import org.gradle.internal.os.OperatingSystem
 buildscript {
     repositories {
         maven {
-            url 'http://maven.ej-technologies.com/repository'
+            url 'https://maven.ej-technologies.com/repository'
```

Comment:
This is a very good point, and @koppor and me don't see any downside for using https. Despite that, this has nothing to do with the rest this PR, so I would like you to open another PR for this.